core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -1,40 +1,38 @@
 # frozen_string_literal: true
 
-require 'rails/generators'
 require 'rails/version'
-require_relative 'install_generator/bundler_context'
-require_relative 'install_generator/support_solidus_frontend_extraction'
-require_relative 'install_generator/install_frontend'
+require 'rails/generators'
+require 'rails/generators/app_base'
 
 module Solidus
   # @private
-  class InstallGenerator < Rails::Generators::Base
+  class InstallGenerator < Rails::Generators::AppBase
```

Comment:
I'm a bit concerned that `Rails::Generators::AppBase` is a bit private. It makes sense to use it in our context, but [Rails guides](https://guides.rubyonrails.org/generators.html) always reference `Rails::Generators::Base`.

The risk is mitigated thanks to the new CI job, though.

Reply:
AppBase is the one used by templates, another way would be to have a template file for the whole installation process and call it from a `Rails::Generators::Base` but I don't think it will gain us much. Also it's seems unlikely that AppBase is going away in any new Rails version. Open to rethink this though 👍

Reply:
Fair enough! The new CI job will tell us if something is broken.